druntime/src/core/time.d

```diff
@@ -3925,7 +3925,10 @@ version (CoreUnittest) const(char)* numToStringz()(long value) @trusted pure not
 }
 
 
-import core.internal.traits : AliasSeq;
+// Workaround for <https://github.com/dlang/dmd/issues/18223>.
+// A selective import of `AliasSeq` happens to bleed through and causes symbol clashes downstream.
+// import.core.internal.traits : AliasSeq;
+import core.internal.traits;
```

Comment:
can we do a `version(none)` and a `@@@ BUG @@@` reference here like we do with `@@@DEPRECATED_2.120@@@` for DMD?

Reply:
Did I do this right?

Reply:
I don't know if there is a right, that works